acdh-schema.owl

```diff
@@ -3345,7 +3376,7 @@ Note that though a Project might (and usually will) use or produce data, it is N
         <rdfs:subClassOf>
             <owl:Restriction>
                 <owl:onProperty rdf:resource="https://vocabs.acdh.oeaw.ac.at/schema#isTitleImageOf"/>
-                <owl:maxCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:maxCardinality>
+                <owl:minCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">0</owl:minCardinality>
```

Comment:
Theoretically we could leave this implicit, since it is the same as saying "class is in domain of this property", as we do with all other classes with cardinality `0-n` on certain properties. However, the more you make explicit in OWL, the better, I would say.

Reply:
So, basically we should just remove line 3348 `<owl:maxCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:maxCardinality>`